app/controllers/seatmaps_controller.rb

```diff
@@ -24,7 +24,7 @@ def create                    # must be reached by 'post' due to file upload
   def update
     return destroy if params[:commit] =~ /delete/i
     @seatmap = Seatmap.find params[:id]
-    if @seatmap.update(seatmaps_update_params)
+    if @seatmap.update_attributes(seatmaps_update_params)
```

Comment:
This was a bad change initially. [Golden repo](https://github.com/armandofox/audience1st/blob/main/app/controllers/seatmaps_controller.rb#L30) used update_attributes